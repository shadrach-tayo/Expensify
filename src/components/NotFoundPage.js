import React from 'react';
import { NavLink } from 'react-router-dom';

const NotFoundPage = () => (
  <div>
    this is the NotFoundPage <NavLink to="/">Go home</NavLink>
  </div>
);

export default NotFoundPage;
