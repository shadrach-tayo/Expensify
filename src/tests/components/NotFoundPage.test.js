import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import NotFoundPage from '../../components/NotFoundPage';

// react test renderer
test('should render Header correctly', () => {
  const wrapper = shallow(<NotFoundPage />);
  expect(toJSON(wrapper)).toMatchSnapshot();
});
