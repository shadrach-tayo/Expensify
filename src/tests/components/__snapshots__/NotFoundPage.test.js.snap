// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<div>
  this is the NotFoundPage 
  <NavLink
    activeClassName="active"
    to="/"
  >
    Go home
  </NavLink>
</div>
`;
